presto-main/src/main/java/com/facebook/presto/sql/planner/plan/Patterns.java

```diff
@@ -164,7 +164,7 @@ private Patterns() {}
 
     public static class Aggregation
     {
-        public static Property<AggregationNode, List<Symbol>> groupingKeys()
+        public static Property<AggregationNode, List<Symbol>> groupingColumns()
         {
             return property("groupingKeys", AggregationNode::getGroupingKeys);
```

Comment:
Is it intentional that the `Patterns#groupingKeys` got renamed even though the implementation did not change?